examples/deploy_service/outputs.tf

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
Don't need all outputs for all examples, just actually useful things (ex. endpoint for service).

Reply:
Done.